### PR TITLE
minor corrections - README.md

### PR DESCRIPTION
**Pull request by geekcom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,11 +27,7 @@ with the bundled [AggregateRoot](https://github.com/prooph/event-sourcing/blob/m
...
-#Used Third-Party Libraries
+# Used Third-Party Libraries
 
 - Uuids of the AggregateChangedEvents are generated with [ramsey/uuid](https://github.com/ramsey/uuid)
-- Assertions are performed by [beberlei/assert](https://github.com/beberlei/assert)
-
-
-
-
+- Assertions are performed by [beberlei/assert](https://github.com/beberlei/assert)
```
Comment (prolic) [inline README.md]: New line at end of file missing

Comment (geekcom) [inline README.md]: revised and ok

**Comment by prolic** [comment]
@geekcom why did you close this PR?

**Comment by geekcom** [comment]
So sorry, it's ok.

**Comment by prolic** [comment]
thanks!

**Pull request merged**